docs/deploy/setup-iomesh.md

```diff
@@ -84,11 +84,14 @@ spec:
 ```
 
 #### Mount Type
+In `mixed-flash` deployment node, IOMesh provides 2 mount types:
 
-IOMesh provides 2 mount types now:
+- `cacheWithJournal`: used for performance tier of storage pool. It **MUST** be a partitionable block device. IOMesh will partition the device into 2 partitions: one for `journal` and another for `cache`. Either `SATA` or `NVMe` SSD is recommended.
```

Comment:
`IOMesh will partition the device into 2 partitions` -> `Two partitions will be created`